src/types/Status.ts

```diff
@@ -1 +1,7 @@
 export type Status = 'all' | 'active' | 'completed';
+
+export enum StatusValues {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
```

Comment:
```suggestion
export enum StatusValues {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
```
Enum properties should be PascalCase, not upper case